CONTRIBUTING.md

```diff
@@ -14,8 +14,8 @@ Here's a few tips as you go along:
 # Translations
 
 * All articles may be translated into any language of interest.
-* Instead of translating from scratch, consider starting with [machine-translated text](https://www.deepl.com/translator) and revising from there.
-Remove "Translated by DeepL" in the final text.
+* Instead of translating from scratch, contact [@nysenthil] to get an IBM Watson machine translation as a starting point and revising from there.
```

Comment:
```suggestion
* Instead of translating from scratch, consider contacting [@nysenthil] to get an IBM Watson machine translation as a starting point and revising from there.
```

I think retaining the word 'consider' is good to make it clear this is optional - maybe I'm overthinking it, but a potential contributor may be put off thinking they aren't allowed to manually translate from scratch.

Reply:
Good idea